### DISCUSSION TITLE
Could not load file or assembly 'CefSharp.Core.Runtime

### DISCUSSION THREAD

**Discussion by Raghava56** [cefsharp/CefSharp#3739] [General]
Here is the scenario.
I installed the CefSharp NuGet package in one of the class library projects and not in Main Wpf Project (Host project).
In Release mode the issue is **Could not load file or assembly 'CefSharp.Core.Runtime, Version=91.1.230.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138' or one of its dependencies. The system cannot find the file specified.#   at CefSharp.CefSettingsBase..ctor()**
I added things in the projects
1.`<CefSharpAnyCpuSupport>true</CefSharpAnyCpuSupport>` in both csproj files (one class library and Host project)
2. #if ANY CPU
           //Only required for PlatformTarget of AnyCPU
           CefRuntime.SubscribeAnyCpuAssemblyResolver();
        #endif
 Added this in the Startup method.

Please help me regarding this.

Thank in advance

**Comment by amaitland** [comment]
Please use discussions to ask questions. Also please don't open any additional issues as you've already have  #3737

> #if ANY CPU
//Only required for PlatformTarget of AnyCPU
CefRuntime.SubscribeAnyCpuAssemblyResolver();
#endif

Remove the `#if/#endif`
